apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -1,12 +1,73 @@
 {{title}}
 
-`Google.Cloud.Spanner.Data` is the ADO.NET provider for Google Cloud Spanner. It is the recommended
-package for regular Spanner database access from .NET.
+`Google.Cloud.Spanner.Data` is the ADO.NET provider for Cloud Spanner. It is the recommended
+package for regular Cloud Spanner database access from .NET.
 
-The [Google.Cloud.Spanner.Admin.Database.V1](../Google.Cloud.Spanner.Admin.Database.V1/)
-and [Google.Cloud.Spanner.Admin.Instance.V1](../Google.Cloud.Spanner.Admin.Instance.V1/) packages 
-should be used for database and instance administration respectively, such as creating or deleting
-databases and instances.
+The [Google.Cloud.Spanner.Admin.Instance.V1](../Google.Cloud.Spanner.Admin.Instance.V1/) package
+should be used for Cloud Spanner instance administration, such as creating or deleting instances.
 
 {{installation}}
 
+{{auth}}
+
+# Getting started
+
+Operations generally follow ADO.NET conventions.
+However Cloud Spanner does not support DDL (eg. "INSERT INTO ..."). Therefore you should
+use the corresponding API on SpannerConnection to create ADO command objects instead
+of setting CommandText manually.
+
+See examples below:
+
+# Sample code
+Once you have created your Google Cloud Project and Spanner Instance using the web console,
+you can start using the ADO.NET provider to create and modify a Cloud Spanner database.
+
+## Creating a Database and Table
+
+[!code-cs[](obj/snippets/Google.Cloud.Spanner.Data.SpannerOverviews.txt#CreateDatabaseAsync)]
+
+## DataAdapter support (.NET 4.5+ only)
+
+Cloud Spanner supports a limited DataAdapter that provides basic CRUD operations on any table.
+Create a SpannerDataAdapter with the target table and its primary keys.  You may replace any
+commands on the SpannerDataAdapter with your own custom commands.
```

Comment:
settled for:
You may replace any
commands provided by SpannerDataAdapter with your own custom commands